gluten-core/src/main/scala/io/glutenproject/GlutenPlugin.scala

```diff
@@ -77,13 +77,15 @@ private[glutenproject] class GlutenDriverPlugin extends DriverPlugin with Loggin
   }
 
   private def showGlutenBuildInfo(): Unit = {
-    val veloxInfo = if (BackendsApiManager.veloxBackend) {
-      s"""
-        |Velox branch: $VELOX_BRANCH
-        |Velox revision: $VELOX_REVISION
-        |Velox revision time: $VELOX_REVISION_TIME""".stripMargin
-    } else {
-      ""
+    val backendInfo = BackendsApiManager.getBackendName.toLowerCase() match {
+      case "velox" => s"""
```

Comment:
Nit:
`BackendsApiManager.veloxBackend` / `BackendsApiManager.chBackend` can be directly used to recognize the current backend, which is added recently. Thanks!

Reply:
@PHILO-HE thanks, fixed.